crates/aptos-genesis/src/builder.rs

```diff
@@ -451,8 +453,11 @@ impl Builder {
             .map(|i| self.generate_validator_config(i, &mut rng, &template))
             .collect::<anyhow::Result<Vec<ValidatorNodeConfig>>>()?;
 
+        // TODO get list of initial accounts?
+        let initial_accounts = vec![];
+
         // Build genesis
-        let (genesis, waypoint) = self.genesis_ceremony(&mut validators, root_key.public_key())?;
+        let (genesis, waypoint) = self.genesis_ceremony(&mut validators, root_key.public_key(), initial_accounts)?;
```

Comment:
Instead of this, I think the initial accounts should be part of Layout and passed into GenesisConfiguration.